### PR TITLE
Add EIP: Deterministic AA wallet

### PR DESCRIPTION
**Pull request by jaehunkim**
We propose a non-custodial interface for service providers to provide AA accounts.

### CODE REVIEW

**Comment by lightclient** [comment]
What part of this EIP proposes a change to the core protocol?

File: EIPS/eip-7710.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+---
+eip: 7710
```
Comment (abcoathup) [inline EIPS/eip-7710.md:2]:
```suggestion
eip: TBA
```
EIP/ERC numbers are assigned by editors/associates.  Please don't self assign, 7710 was already assigned to an ERC.

If this is an ERC (application layer) please close this PR and open a new PR in the ERC repo.

**Comment by abcoathup** [comment]
@jaehunkim can you answer lightclient's question.  An EIP number can't be assigned until it is determined whether this is an EIP or an ERC which should be in the ERC repo.

>What part of this EIP proposes a change to the core protocol?

**Comment by jaehunkim** [comment]
@abcoathup Thank you for the reminder. As @lightclient mentioned above, this issue seems more suitable for the ERC. I will move the PR.

**Pull request closed without merging**